cogip/mcu-firmware
# Clean riot patches

As they all have been applied in COGIP RIOT repository[1].

All examples and applications can be built for their target architecture and for cogip-native board.

Testing procedure:
```shell
$ make -j
[...]
$ echo $?
0

$ make -j BOARD=cogip-native
[...]
$ echo $?
0

$ arm-none-eabi-gcc --version
arm-none-eabi-gcc (Arm GNU Toolchain 13.2.rel1 (Build arm-13.7)) 13.2.1 20231009
Copyright (C) 2023 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

$ gcc --version
gcc (Ubuntu 12.3.0-1ubuntu1~22.04) 12.3.0
Copyright (C) 2022 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

[1] https://github.com/cogip/RIOT

Fix #148 #144
